xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -40,7 +41,76 @@
 #include "pvr/recordings/PVRRecordings.h"
 #include "pvr/timers/PVRTimerInfoTag.h"
 
-using namespace PVR;
+namespace PVR
+{
+
+class CEpgUpdateRequest
```

Comment:
Why not define this in the header file?

Reply:
Because it's an internal class and I always try to reduce type visibility to a minimum.